packages/web-pkg/src/components/sidebar/types.ts

```diff
@@ -8,6 +8,7 @@ export interface Panel {
   title: string
   component: Component
   componentAttrs: any
+  componentHandlers: any
```

Comment:
```suggestion
  componentListeners: any
```
I know, ultra nitpick. But from my understanding the handler is only the callback, while the listener maps a handler to a certain event, so what you're adding here are listeners, not handlers. Do you agree, or is that flawed?

Since this is not "some application" but `web-pkg` (which we want to publish at some point in the future) I'd like to be strict with naming.